composer.json

```
@@ -20,7 +20,7 @@
         "nette/utils": "^3.2",
         "nikic/php-parser": "4.12.0",
         "phpstan/phpdoc-parser": "^0.5.5",
-        "phpstan/phpstan": "0.12.96",
+        "phpstan/phpstan": "^0.12.96",
```

we need to pinned version:

```suggestion
        "phpstan/phpstan": "0.12.97"
```

as that may broke rector, we need to always verify that the version of phpstan and php-parser works.